terminology.md

```diff
@@ -421,6 +421,38 @@ This means demonstrating both (1) that it addresses a significant developer need
 
 Additionally, in a shorter-term sense, it could overwhelm developers to have too many syntax-heavy features ship in too narrow of a timeframe.
 
+## In-Band
+
+### Definition
+
+Used mainly in opposition to [Out of band](#out-of-band), denoting that all configuration of a feature or a behavior is done within the context of ECMASCript source.
+
+### Example
+
+- The stage 3 Import assertions proposal assert pragma
+- The "use strict" directive, an in-band way to define the semantics of the script contained in the scope where it is used.
```

Comment:
```suggestion
- The "use strict" directive, an in-band way to enable strict mode for some portion of code.
```

Also I would probably put this first since readers will be much more familiar with it.